Packs/Twitter/Integrations/Twitterv2/Twitterv2.py

```diff
@@ -0,0 +1,360 @@
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    client.tweet_search('Twitter', None, None, 10, None)
+    message = 'ok'
+    return message
+
+
+def date_to_iso_format(date: str) -> str:
+    """ Retrieves date string or relational expression to iso format date.
+        Args:
+            date: str - date or relational expression.
+        Returns:
+            A str in ISO format.
+    """
+    date = dateparser.parse(date)
+    date = date.strftime("%Y-%m-%dT%H:%M:%SZ") if date else ''
+    return date
+
+
+def create_human_readable_search(dict_list: list[dict]) -> list[dict]:
+    """Gets list of dictionaries and creates a human readable from it.
+    Args:
+        dict_list: list[dict] -  A list of dictionaries.
+    Returns:
+        human readable: list[dict].
+    """
+    list_dict_response: list = []
+    for dict_value in dict_list:
+        list_dict_response.append(
+            {
+                "Tweet ID": dict_value.get("id", {}),
+                "Text": dict_value.get("text", ""),
+                "Created At": dict_value.get("created_at", {}),
+                "Author Name": dict_value.get("author", {}).get("name"),
+                "Author Username": dict_value.get("author", {}).get("username"),
+                "Likes Count": dict_value.get("public_metrics", {}).get("like_count"),
+                "Attachments URL": [
+                    item.get("url", "") for item in dict_value.get("media", [])
+                ]
+                if dict_value.get("media", [])
+                else dict_value.get("media", []),
+            }
+        )
+    return list_dict_response
+
+
+def twitter_tweet_search_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    """ Gets args and client and returns CommandResults of Tweets according to the reqested search.
+        Args:
+            client: client -  A Twitter client.
+            args: Dict - The function arguments.
+        Returns:
+            A list of CommandResults with Tweets data according to to the reqest.
+    """
+    headers = ['Tweet ID', 'Text', 'Created At', 'Author Name', 'Author Username', 'Likes Count', 'Attachments URL']
+    query = argToList(args.get('query'))
+    start_time = date_to_iso_format(args.get('start_time', '')) if args.get('start_time', None) else None
+    end_time = date_to_iso_format(args.get('end_time', '')) if args.get('end_time', None) else None
+    limit = arg_to_number(args.get('limit', 50))
+    next_token = args.get('next_token', None)
+    raw_response, result, next_token = client.tweet_search(query, start_time, end_time, limit, next_token)
+    dict_to_tableToMarkdown = create_human_readable_search(result)
+    human_readable = tableToMarkdown("Tweets search results:", dict_to_tableToMarkdown,
+                                     headers=headers, removeNull=False)
+    command_results = []
+    if next_token:
```

Comment:
Change the order as we did before 
First the results itself then the check for next_token